doc/source/ray-libraries.rst

```diff
@@ -151,6 +164,10 @@ XGBoost is a popular gradient boosting library for classification and regression
     :class: inline-figure
     :height: 30
 
+.. |ptl| image:: images/pytorch_lightning_small.png
```

Comment:
is there a picture you should be adding?

Reply:
The png file is already in the images directory.